src/main.js

```diff
@@ -51,7 +52,7 @@ let load = Loader({
 
         },
         sentryDSN: async (project) => {
-          let key = await this.sentryManager.projectDSN(project);
+          let key = await sentryManager.projectDSN(project);
```

Comment:
I thought we made this change yesterday?

Reply:
Oh no, apparently I didn't land that!  I pushed to origin/master :(